project/Build.scala

```diff
@@ -79,13 +81,13 @@ object Build extends Build {
     .settings(sbtassembly.Plugin.assemblySettings: _*)
     .settings(playArtifactSettings: _*)
     .settings(fiddlyExtraAssemblySettingsForExport: _*)
-    .settings(assemblyMergeSettings: _*)
 
   def fiddlyExtraAssemblySettingsForExport = Seq(
     mergeStrategy in assembly <<= (mergeStrategy in assembly) { (old) => {
       case "version.txt" => MergeStrategy.first
       case "play.plugins" => MergeStrategy.first
       case "logback.xml" => MergeStrategy.first
+      case "play/core/server/ServerWithStop.class" => MergeStrategy.first
```

Comment:
It's just moved from the `assemblyMergeSettings` below, tidier following the refactoring.